tooling/target.yml

```diff
@@ -8,6 +8,8 @@ networks:
 
 services:
   knotx:
+    environment:
+      - TARGET_MOCK_DOMAIN=${TARGET_MOCK_DOMAIN}
```

Comment:
Could you please check if the default value here will work?
```
- TARGET_MOCK_DOMAIN=${TARGET_MOCK_DOMAIN:-host.docker.internal}
```
in that way, no one will be forced to add this evn locally.

Reply:
it works, thanks